client/src/common/hooks/useSearch.ts

```diff
@@ -35,6 +42,28 @@ const useSearch = () => {
     },
   })
 
+  const [getExtrinsicByHash] = useLazyQuery(QUERY_EXTRINSIC_BY_HASH, {
+    onCompleted: (data) => {
+      if (data.extrinsics.length > 0) {
```

Comment:
does it mean if we have two extrinsics with the same hash it will navigate to the first one? I think we discussed that it should show both (a list)

Reply:
Yeah, this first PR is taking into account only showing the first extrinsic, in the next one, I'll add the result page and the necessary validation. Just want to test that case better before submitting the PR

Reply:
make sense